GitHub_Learning/Feature/Onboarding/View/OnboardingLayout.swift

```diff
@@ -7,15 +7,82 @@
 //
 
 import UIKit
+import SnapKit
 
-class OnboardingLayout : UIView {
+class OnboardingLayout: UIView {
+    
+    private lazy var onboardingStack: UIStackView = {
+        let stackView = UIStackView(arrangedSubviews: [pagesCollectionView, pagesControl, nextButtonStack])
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.axis = .vertical
+        return stackView
+    }()
+    
+    private lazy var nextButtonStack: UIStackView = {
+        let stackView = UIStackView(arrangedSubviews: [nextButton])
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.isLayoutMarginsRelativeArrangement = true
+        stackView.layoutMargins = UIEdgeInsets(top: 0, left: 20, bottom: 0, right: 20)
+        return stackView
+    }()
+    
+    let pagesCollectionView: UICollectionView = {
+        let cv = UICollectionView(frame: .zero, collectionViewLayout: UICollectionViewFlowLayout())
+        cv.translatesAutoresizingMaskIntoConstraints = false
+        cv.showsHorizontalScrollIndicator = false
+        cv.allowsSelection = false
+        cv.isPagingEnabled = true
+        cv.backgroundColor = .clear
+        cv.register(OnboardingCell.self, forCellWithReuseIdentifier: OnboardingCell.onboardingCellId)
+        let layout = (cv.collectionViewLayout as? UICollectionViewFlowLayout)
+        layout?.scrollDirection = .horizontal
```

Comment:
Misto pretypovani si vytvor flow layout, nastav si ho a pak ho predej do collection view. Tj dej to na zacatek tohoto setupu.